dindctl

```diff
@@ -46,13 +46,14 @@ SUPERVISOR_EXTRA_MOUNTS=""
 DIST_MOUNTED="false"
 DIND_IMAGE="resin-supervisor-dind"
 PRELOADED_IMAGE=""
+OPTIMIZE="true"
```

Comment:
What do you think of an option where we run the supervisor directly from the coffee-script files in dindctl without needing to separately recompile? It would make debugging even easier, especially if sourcemaps are giving us the correct line numbers